core/src/sigverify_stage.rs

```diff
@@ -74,24 +74,43 @@ pub struct DisabledSigVerifier {}
 
 #[derive(Default)]
 struct SigVerifierStats {
-    recv_batches_us_hist: histogram::Histogram, // time to call recv_batch
-    verify_batches_pp_us_hist: histogram::Histogram, // per-packet time to call verify_batch
-    discard_packets_pp_us_hist: histogram::Histogram, // per-packet time to call verify_batch
-    dedup_packets_pp_us_hist: histogram::Histogram, // per-packet time to call verify_batch
-    batches_hist: histogram::Histogram,         // number of packet batches per verify call
-    packets_hist: histogram::Histogram,         // number of packets per verify call
+    /// The time (us) spent calling recv_batch() to get PacketBatches into the stage
+    recv_batches_us_hist: histogram::Histogram,
+    /// The per packet mean (per loop) time (us) spent performing signature verification
+    verify_batches_pp_us_hist: histogram::Histogram,
+    /// The per packet mean (per loop) time (us) spent discarding packets after deduplication
+    discard_packets_pp_us_hist: histogram::Histogram,
+    /// The per packet mean (per loop) time (us) spent deduplicating packets
+    dedup_packets_pp_us_hist: histogram::Histogram,
+    /// The number of batches (per loop)
+    batches_hist: histogram::Histogram,
+    /// The number of packets (per loop)
+    packets_hist: histogram::Histogram,
+    /// The number of Deduper saturations
```

Comment:
Might be nice to call out what this means. E.g. "number of times the deduper was reset due to false positive rate exceeding allowed threshold"

Reply:
Yeah, this did occur to me while writing. But in the same vein as
> I'm always a little leary of calling out specific function/variable names in comments

my thought was leaving more detailed explanation with `Deduper` and that someone could reference that. Wdyt about something like:
> /// The number of Deduper saturations, see Deduper for more details on what a saturation is

Reply:
Yeah, that sounds good to me